src/test/unit/engine/LinkTest.cpp

```diff
@@ -500,6 +500,13 @@ class TestRegionBase : public RegionImpl, Serializable {
   // De-serialize state. Must be called from deserializing constructor
   void deserialize(BundleIO &bundle) override {}
 
+  
+  bool operator==(const RegionImpl &other) const override { return true; }
```

Comment:
ditto, NTA_THROW. even though this is a test, so the always-true compare might be acceptable here if needed